composer.json

```diff
@@ -31,10 +31,10 @@
   },
   "require": {
     "php": ">=5.6",
-    "cakephp/cakephp": "~3.2"
+    "cakephp/cakephp": "~3.6"
```

Comment:
Bumping to 3.6 is not necessary. The updated methods exist in 3.5 too so please change this to `^3.5`.